ghmirror/core/mirror_requests.py

```diff
@@ -105,8 +105,8 @@ def online_request(method, url, auth, data=None):
     resp.headers['X-Cache'] = 'ONLINE_MISS'
     # Caching only makes sense when at least one
     # of those headers is present
-    if any(['ETag' in resp.headers,
-            'Last-Modified' in resp.headers]):
+    if resp.status_code == 200 and any(['ETag' in resp.headers,
+                                        'Last-Modified' in resp.headers]):
```

Comment:
this change belongs to a different commit...

Reply:
its in a separate commit now. want it in a separate PR?

Reply:
No, looks good, thanks!